app/src/main/java/com/duycuong/weather/utils/TimeUtils.java

```diff
@@ -11,6 +12,22 @@
     public static String covertTime(String format, long time) {
         SimpleDateFormat simpleDateFormat =
                 new SimpleDateFormat(format, Locale.getDefault());
-        return simpleDateFormat.format(time * 1000);
+        return simpleDateFormat.format(time*1000);
+    }
+
+    public static long ConvertDateToTimeStamp(String ngay){
+        final Date ngayquydoi;
+        long timestamp=0;
+        SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yyyy");
+        try
+        {
+            ngayquydoi = sdf.parse(ngay);
+            timestamp=ngayquydoi.getTime();
+        }

```

Comment:
ERROR: '}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).